Wrappers/Python/cil/optimisation/operators/SparseFiniteDifferenceOperator.py

```diff
@@ -90,7 +90,6 @@ def sum_abs_col(self):
         return ImageData(res)
 
 if __name__ == '__main__':
-    from cil.framework import ImageGeometry
     M, N= 2, 3
     ig = ImageGeometry(M, N)
     arr = ig.allocate('random_int')
```

Comment:
This whole main/entry point code should go.

Reply:
I presume it needs to be moved to tests?

Reply:
It should already be covered in tests. But Leave this for a followup PR.

Reply:
moved to #1903